bundle.Dockerfile

```diff
@@ -4,9 +4,9 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=kogito-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=alpha,1.x
-LABEL operators.operatorframework.io.bundle.channel.default.v1=1.x
+LABEL operators.operatorframework.io.bundle.package.v1=rhpam-kogito-operator
```

Comment:
won't this be overriden by https://github.com/kiegroup/kogito-operator/blob/1.13.x-blue/rhpam-image-bundle.yaml ?

Reply:
yes, will revert and update only for release.